src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticListenerExtensions.cs

```diff
@@ -10,7 +10,7 @@ public static class DiagnosticListenerExtensions
         public static IDisposable SubscribeWithAdapter(this DiagnosticListener diagnostic, object target)
         {
             var adapter = new DiagnosticSourceAdapter(target);
-            return diagnostic.Subscribe(adapter, (Predicate<string>)adapter.IsEnabled);
+            return diagnostic.Subscribe(adapter, s => adapter.IsEnabled(s));
```

Comment:
What's up with this change?

Reply:
@natemcmaster did these as part of #70. I undid them for now since I don't the reason to change it. If we need these for any reason I can look into it as part of a new work item.

Reply:
.Subscribe has overloads in netcoreapp2.0/net461 that were not in ns1.3. The call was ambiguous so needed to be changed to avoid compiler errors.

See https://github.com/aspnet/EventNotification/pull/72#discussion_r120981319.